### ISSUE TITLE
1.3 beta 4: "import *" not working

### ISSUE THREAD

**Issue by ffxsam** [meteor/meteor#5994]
```
import {createStore, combineReducers} from 'redux';
import * as reducers from './reducers';

export default todoApp = combineReducers(reducers)
```

This yields an error, though from what I've read, `import * as xyz from 'module'` is valid ES6 syntax.

**Comment by benjamn** [comment]
This will be fixed in the next beta, thanks to 3a9907aa3acbee84aaa5759bfcb241bf1fe8b229.

**Comment by ffxsam** [comment]
Awesome, thanks!
